src/components/BreadCrumb/BreadCrumb.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div id="breadcrumb">
+    <v-container>
+      <v-row no-gutters>
+        <v-col cols="auto">
+          <v-btn
+            id="breadcrumb-back-btn"
+            class="back-btn"
+            exact
+            :href="backUrl"
+            icon small
+            :disabled="breadcrumbs.length <= 1"
+          >
+            <v-icon color="#38598a">mdi-arrow-left</v-icon>
+          </v-btn>
+        </v-col>
+
+        <v-divider class="mx-3" color="white" vertical />
+
+        <v-col cols="auto" class="breadcrumb-col">
+          <v-breadcrumbs :items="breadcrumbs" class="pa-0 ma-0">
+            <template v-slot:divider>
+              <v-icon color="white" class="mx-n2">mdi-chevron-right</v-icon>
+            </template>
+
+            <v-breadcrumbs-item
+              slot="item"
+              slot-scope="{ item }"
+              exact
+              :to="item.to"
+              :href="item.href"
+            >
+              <span class="breadcrumb-text" :class="{'active-crumb': isActiveCrumb(item)}">{{ item.text }}</span>
+            </v-breadcrumbs-item>
+          </v-breadcrumbs>
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { BreadcrumbIF } from '@/interfaces'
+
+@Component({})
+export default class BreadCrumb extends Vue {
+  /** Breadcrumb meta data.
+   * Example breadcrumb:
+   * {
+   *  text: My Dashboard,
+   *  disabled: false,
+   *  exact: true,
+   *  to: { name: Routes.DASHBOARD }
+   * }
+   * */
+  @Prop({ default: () => [] })
+  readonly breadcrumbs: Array<BreadcrumbIF>
+
+  /**
+   *  The url tied to the breadcrumb back button.
+   *  Cross application requires href.
+   *  Returns to previous route when navigating locally.
+   */
+  private get backUrl (): string {
+    const routeIndex = this.breadcrumbs.findIndex(item => item.to?.name === this.$route?.name)
```

Comment:
What if no match is found?

Reply:
Then something is broken :) 
Without the route the user is most likely not in the application any longer.
I suppose it is a requirement to have a `to` on local starting point, so we can count on that.
`to` is optional according to the interface as we will be passing external crumbs which rely on a href, but locally (in our ui's) we will always have/need `to` routes for local navigation. 

So tldr: this check finds our starting point, regardless of how many href crumbs before our local starting point and subsequently tracks any deeper navigation through the `to`. Although we don't have too many implementations of deeper navigation outside the filings ui.

Reply:
OK. So we leave it up to the user of this component to configure it correctly or else.

Is there any risk of the configuration being messed up under normal running conditions (since I know it's dynamic)?

Reply:
Well, it's no real risk. 
It will simply exist but not do anything. 
For example, if a user does not pass href or to values, the navigation will not go anywhere. 
If they don't pass in any text, there will be crumb on display and no link etc. 

So essentially, no real risk, any persons implementing this can see the example crumb on the prop or refer to the interface. Functionally it just won't do what they want it to do until they pass it a value.

Reply:
I buy that explanation - thanks.

One thing to watch for is that an unhandled exception can cause a method/component to stop running/displaying. This could be serious depending on what gets broken (imagine something in App or TodoList or ...). In this case I think the risk is low: this component may "crash" but I don't think it will take anything down with it, so I think we're good.